c2cgeoportal/views/layers.py

```diff
@@ -362,7 +376,16 @@ def metadata(self):
         if not layer.public and self.request.user is None:
             raise HTTPForbidden()
 
-        return self._metadata(str(layer.geo_table), layer.exclude_properties)
+        # exclude the columns used to record the last features update
+        exclude = layer.exclude_properties if layer.exclude_properties else ""
+        last_update_date = self.settings.get("last_update_date_column", False)
+        if last_update_date:
+            exclude += "," + last_update_date
```

Comment:
layer.exclude_properties is a string: `a,b,c`
i'm adding an element to this list with `"," + last_update_date` to have `a,b,c,d`